tests/integration/components/public/event-map-test.js

```diff
@@ -0,0 +1,35 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('g-map', 'Integration | Component | g map', {
```

Comment:
See other integration tests to see which `moduleForComponent` method is used and how it's used. Modify accordingly.